roles/lib_openshift/src/test/unit/conftest.py

```diff
@@ -0,0 +1,55 @@
+import inspect
+
+import pytest
+
+from lib_openshift.library import *  # noqa: F401,F403
```

Comment:
:+1: I'll take a look at updating, I rabbit holing down that path and went this route out of convenience.